CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Changelog
 
+## [0.22.13](https://github.com/brave/browser-laptop/releases/tag/v0.22.13dev)
+
+ - Added a new recovery dialog with information when detecting corrupted seed. ([#13424](https://github.com/brave/browser-laptop/issues/13424))
+ - Added U2F support for various websites. ([#518](https://github.com/brave/browser-laptop/issues/518))
+ - Added "no thanks" to Brave Payment notifications. ([#12758](https://github.com/brave/browser-laptop/issues/12758))
+ - Added "always block" option to autoplay. ([#9755](https://github.com/brave/browser-laptop/issues/9755))
+ - Added "in progress" status under Brave Payments when contribution in progress. ([#13423](https://github.com/brave/browser-laptop/issues/13423))
+ - Added keyboard shortcuts to re-order tabs. ([#11310](https://github.com/brave/browser-laptop/issues/11310))
+ - Added EV certificate status in URL bar. ([#791](https://github.com/brave/browser-laptop/issues/791))
+ - Added new look for tab previews when hovering. ([#12454](https://github.com/brave/browser-laptop/issues/12454))
+ - Optimized opening new windows. ([#12437](https://github.com/brave/browser-laptop/issues/12437))
+ - Updated HTTPS Everywhere. ([#11598](https://github.com/brave/browser-laptop/issues/11598))
+ - Disabled WebUSB API. ([#13374](https://github.com/brave/browser-laptop/issues/13374))
+ - Fixed printing issues under Windows. ([#13546](https://github.com/brave/browser-laptop/issues/13546))
+ - Fixed scrolling sometimes becoming stuck. ([#13580](https://github.com/brave/browser-laptop/issues/13580))
+ - Fixed pinned site contributions so they are excluded from statistical voting. ([#13431](https://github.com/brave/browser-laptop/issues/13431))
+ - Fixed twitch not being added into the ledger table. ([#13597](https://github.com/brave/browser-laptop/issues/13597))
```

Comment:
nit: twitch -> Twitch
Capitalization for branded items per what the brand defines.  Ditto in references below.